modules/create-a-kubeletconfig-crd-to-edit-kubelet-parameters.adoc

```diff
@@ -61,6 +61,28 @@ spec:
     maxPods: 500
 ----
 +
+The rate at which the kubelet talks to the API server depends on qps and burst
+values. The default values, `5` for `kubeAPIQPS` and `10` for `kubeAPIBurst`,
+are good enough if there are limited pods running on each node. Updating the
```

Comment:
Suggestion - this can be active voice, "If there are enough CPI and memory, we recommend..."

Although, I checked the existing docs and we usually use the passive voice when making a recommendation.

Reply:
@lamek Thanks! Yes, we avoid saying "we recommend".